### DISCUSSION TITLE
Enable negative offsets similar to Prometheus

### DISCUSSION THREAD

**Discussion by andreeadima96** [thanos-io/thanos#4775] [Questions & Answers]
Regarding  _--enable-feature=promql-negative-offset_, I see it added on  [Changelog](https://thanos.io/tip/thanos/changelog.md/#added)  at Unreleased, Added section.

When will it be released?

Thank you!

**Comment by yeya24** [comment]
Please see https://github.com/thanos-io/thanos/issues/4022.
Current main branch has a bug on this and @metonymic-smokey is actively working on this.

**Reply by yeya24** [reply]
Should be solved by #4786
